index.js

```diff
@@ -6,44 +6,37 @@ const fs = require('fs');
 async function run() {
   try {
     const version = core.getInput('version');
-    const kubeconfig_location="/tmp/output/kubeconfig-"+version+".yaml";
-    console.log(`storing kubeconfig here ${kubeconfig_location}!`); 
+    const ports = core.getInput('ports');
+    const nodeName = 'primary-node'
 
-    await exec.exec('docker', ["run","-d","--privileged","--name=k3s-"+version,
-    "--expose=6443",
-    "-e","K3S_KUBECONFIG_OUTPUT="+kubeconfig_location,
-    "-e","K3S_KUBECONFIG_MODE=666",
-    "-v","/tmp/output:/tmp/output","-p","6443:6443","-p","80:80","-p","443:443","-p","8080:8080",
-    "rancher/k3s:"+version,"server"]);
-    
-    await wait(10000);
+    const kubeconfigPath=`/tmp/output/kubeconfig-${version}.yaml`;
+    console.log(`storing kubeconfig here ${kubeconfigPath}!`);
```

Comment:
is this log intended?

Reply:
It was there originally, it could be moved to be output instead, thoughts

Reply:
It was there originally, it could be moved to be output instead, thoughts